src/segment_owm.go

```diff
@@ -21,6 +21,8 @@ const (
 	LOCATION Property = "location"
 	// UNITS openweathermap units
 	UNITS Property = "units"
+	// CACHETIMEOUT cache timeout
+	CACHETIMEOUT Property = "cachetimeout"
```

Comment:
Mismatch in doc and value `cache_timeout`, would also use the underscore here.

Reply:
my bad... and the schema.json property has also been added on the wrong segment.